helm-service/main.go

```diff
@@ -79,69 +79,81 @@ func gotEvent(ctx context.Context, event cloudevents.Event) error {
 
 	if event.Type() == keptnv2.GetTriggeredEventType(keptnv2.DeploymentTaskName) {
 		deploymentHandler := createDeploymentHandler(url, keptnHandler, mesh)
-		go deploymentHandler.HandleEvent(event)
+		deploymentHandler.HandleEvent(event)
 	} else if event.Type() == keptnv2.GetTriggeredEventType(keptnv2.ReleaseTaskName) {
 		releaseHandler := createReleaseHandler(url, mesh, keptnHandler)
-		go releaseHandler.HandleEvent(event)
+		releaseHandler.HandleEvent(event)
 	} else if event.Type() == keptnv2.GetFinishedEventType(keptnv2.ServiceCreateTaskName) {
-		onBoarder := createOnboarder(keptnHandler, url, mesh)
-		go onBoarder.HandleEvent(event)
+		onboardHandler := createOnboardHandler(url, keptnHandler, mesh)
+		onboardHandler.HandleEvent(event)
 	} else if event.Type() == keptnv2.GetTriggeredEventType(keptnv2.ActionTaskName) {
 		actionHandler := createActionTriggeredHandler(url, keptnHandler)
-		go actionHandler.HandleEvent(event)
+		actionHandler.HandleEvent(event)
 	} else if event.Type() == keptnv2.GetFinishedEventType(keptnv2.ServiceDeleteTaskName) {
 		deleteHandler := createDeleteHandler(url, keptnHandler)
-		go deleteHandler.HandleEvent(event)
+		deleteHandler.HandleEvent(event)
```

Comment:
Why is `go `  removed in these calls? Is the multi-threading aspect of helm-service handled elsewhere?

Reply:
it is handled in the the cloudevents sdk